### PR TITLE
Cleanup styling of invite page.

### PR DESCRIPTION
**Pull request by danawoodman**
- Autofocus input
- Only show input until a member is added
- Make success message clearer
- Fix button sizing
- Make add form inputs larger
- Make page narrower so its more legible
- Clean JS to use const
- Add some more ES6 features
- Use forEach instead of for loop
- Remove "Next" button
- Other general cleanup and formatting

### CODE REVIEW

File: frontend/simple/views/Invite.vue [hunk 1]
```diff
@@ -80,49 +118,64 @@ export default {
...
+        const { contractId } = member
+        const events = await backend.eventsSince(contractId, contractId)
+        const [ contract, ...actions ] = events.map(e => {
           return Events[e.entry.type].fromObject(e.entry, e.hash)
         })
-        let state = contract.toVuexState()
+        const state = contract.toVuexState()
         actions.forEach(action => {
-          let type = action.constructor.name
+          const type = action.constructor.name
```
Comment (taoeffect) [inline frontend/simple/views/Invite.vue]: Hmm, so this section somehow did not get converted to using the new `latestContractState` in `js/state.js`. Let's convert it. Also, at some the `latestContractState` itself should be modified to received a starting position (and use `contractId` as the starting position by default).

**Comment by taoeffect** [comment]
Much improved, nicely done! See my one comment about using `latestContractState` and then it's good to go!

**Comment by taoeffect** [comment]
@danawoodman If you could also, in addressing the comment above, have this be a branch off of the latest `master` (if necessary close & re-open the PR)? That way Travis will kick back in and verify the build.

File: frontend/simple/views/Invite.vue (hunk 1 above)
Comment (danawoodman) [reply frontend/simple/views/Invite.vue]: Should be resolved now

**Pull request closed without merging**